optuna/visualization.py

```diff
@@ -26,7 +26,18 @@
 except ImportError as e:
     _import_error = e
     # Visualization features are disabled because plotly is not available.
-    _available = False
+    available = False
```

Comment:
Should this be 
```suggestion
    _available = False
```
?

Reply:
You're obviously right. Thanks for the catch!